keyboards/40percentclub/foobar/keymaps/qfoobar/config.h

```diff
@@ -0,0 +1,15 @@
+#define PERMISSIVE_HOLD
```

Comment:
```suggestion
#pragma once

#define PERMISSIVE_HOLD
```

Reply:
qfoobar shouldn't be in this, I messed up with my own keymaps, will remove this.